### PR TITLE
Small fix to #3378

### PR DESCRIPTION
**Pull request by pp-mo**
This gets the "missing" tests working.
As per [this comment](https://github.com/SciTools/iris/pull/3378#issuecomment-542153762)

### CODE REVIEW

File: lib/iris/tests/unit/experimental/representation/__init__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# (C) British Crown Copyright 2017, Met Office
```
Comment (stephenworsley) [inline lib/iris/tests/unit/experimental/representation/__init__.py]: Should this copyright be dated 2017?

Comment (pp-mo) [reply lib/iris/tests/unit/experimental/representation/__init__.py]: No.  
I wonder how it is passing `iris.tests.code_coding_standards.TestLicenseHeaders.` ?

**Comment by pp-mo** [comment]
@stephenworsley try that ?

**Review by stephenworsley — APPROVED** [review]
Looks good.

**Pull request merged**